code/style.css

```diff
@@ -47,7 +46,53 @@ input {
   flex: 1;
 }
 
-main {
+/*** INDEX STYLING ***/
+
+.start-main {
+  background-image: url("./assets/lakeview.jpg");
```

Comment:
https://www.w3schools.com/css/css3_object-fit.asp This link should help you style your landing page better 😸